packages/core/core/src/projectPath.js

```diff
@@ -6,7 +6,7 @@ import {relativePath, normalizeSeparators} from '@parcel/utils';
 /**
  * A path that's relative to the project root.
  */
-export opaque type ProjectPath = string;
+export type ProjectPath = string;
```

Comment:
I'm somewhat iffy about the removal of `opaque` here to appease Flow, but it seemed to be the most straightforward solution. Let me know if there are objections to this change.

Reply:
You can use `toProjectPathUnsafe(node.id)` to turn the ids into a `ProjectPath`. I think that is preferable to changing the type to not be opaque, as doing so causes every other `ProjectPath` type across Parcel to be less sound than it was before.